update_dependencies.sh

```diff
@@ -53,12 +57,21 @@ sudo apt install -y \
     dh-python \
     batctl \
     alfred \
+    ros-foxy-octomap \
+    ros-foxy-octomap-msgs \
+    ros-foxy-laser-geometry \
+    ros-foxy-pcl-conversions \
+    ros-foxy-pcl-msgs \
+    ros-foxy-dynamic-edt-3d \
     ros-foxy-gazebo-ros
 
 pip3 install --user pyros-genmsg
 
-wget https://github.com/mavlink/MAVSDK/releases/download/v0.34.0/mavsdk_0.34.0_ubuntu20.04_amd64.deb
-sudo dpkg -i mavsdk_0.34.0_ubuntu20.04_amd64.deb
+curl -LO https://artifactory.ssrc.fi:443/artifactory/debian-release-local/mavsdk_0.41.0_ubuntu20.04_amd64.deb
+sudo dpkg -i mavsdk_0.41.0_ubuntu20.04_amd64.deb
```

Comment:
Tested in simulation but not in the drone AFAIK. Should we test it in the drone before adding it to the build?